nlmod/dims/grid.py

```diff
@@ -317,6 +273,79 @@ def xyz_to_cid(xyz, ds=None, modelgrid=None):
     return cid
 
 
+def get_node_structured(lay, row, col, shape):
+    """Get the node number of a structured grid.
+
+    Parameters
+    ----------
+    lay : int
+        layer number.
+    row : int
+        row number.
+    col : int
+        column number.
+    shape : tuple
+        shape of the model grid.
```

Comment:
Maybe good to add a definition of the shape: (nrow, ncol)

Reply:
it is (nlay, nrow, ncol) I think. Added this to the docstring